### PR TITLE
Jyee/integration sdk

### PR DESCRIPTION
**Pull request by jyee**
Moving the integration SDK documentation from that repo into docs.

### CODE REVIEW

**Comment by jyee** [comment]
See the corresponding PR in integrations-extras: https://github.com/DataDog/integrations-extras/pull/14
And in integrations-core: https://github.com/DataDog/integrations-core/pull/25

**Comment by jyee** [comment]
Outstanding tasks:

- Needs an update for the integration YAML file (I think there was a bug where this wasn't getting copied over initially?)
- Needs an update for how to install locally (see @technovangelist docs at https://github.com/DataDog/documentation/pull/972/files)
- Needs an update so that it follows the installation guide in the docs site listing.

**Comment by technovangelist** [comment]
Ive created a branch called agent5.9. would it be better to merge into that, then merge 59 into master when its released. I have been trying to remember to merge all my changes into 59 (though i think i missed some)

**Comment by jyee** [comment]
@technovangelist I've updated base to `agent5.9`. I think that's a good idea, since it will help us group things together and then release at one time.

I've completed the outstanding tasks I outlined above, so review the *conf.yaml.example* and *installing locally* sections.

File: content/guides/integration_sdk.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+##### `metadata.csv`
+
+The metadata CSV contains a list of the metrics your integration will provide and basic details that will help inform the Datadog web application as to which graphs and alerts can be provided for the metric.
+
+The CSV should include a header row and the following columns:
+
+**`metric_name`** (required): The name of the metric as it should appear in the Datadog web application when creating dashboards or monitors. Often this name is a period delimited combination of the provider, service, and metric (e.g. `aws.ec2.disk_write_ops`) or the application, application feature, and metric (e.g. `apache.net.request_per_s`).
+
+**`metric_type`** (required): The type of metric you are reporting. This will influence how the Datadog web application handles and displays your data. Accepted values are: `count`, `gauge`, or `rate`.
```
Comment (technovangelist) [inline content/guides/integration_sdk.md]: awesome. i think this might be the only spot we define the metric types

**Comment by technovangelist** [comment]
@jyee last i saw this was ready. anything else you need to add?

**Comment by jyee** [comment]
Nope. I think it's good to merge.

**Pull request merged**